core/backend/src/IModelHost.ts

```diff
@@ -399,6 +399,7 @@ export class IModelHost {
   }
 
   private static _briefcaseCacheDir: string;
+  private static _tileCacheDir: string;

```

Comment:
Please don't call this "tile cache".  Maybe "_iModelTemp" for the variable and "imodels-temp" for the directory name.